12-add-tupletools.md

```diff
@@ -53,18 +53,18 @@ The way the `DecayTreeTuple.Decay` is written in in our [minimal DaVinci job](08
 dtt.Decay = '[D*(2010)+ -> (D0 -> K- pi+) pi+]CC'
 ```
 
-means that the configured `TupleTools` will only run on the head of the decay chain, that is, the D*.
+means that the configured `TupleTools` will only run on the head of the decay chain, that is, the `D*`.
```

Comment:
Could you change `D*` to `D*(2010)+`? Then all the `code` particles match the decay descriptor.